cmd/copy.go

```diff
@@ -1258,7 +1258,7 @@ func init() {
 	rootCmd.AddCommand(cpCmd)
 
 	// filters change which files get transferred
-	cpCmd.PersistentFlags().BoolVar(&raw.stripTopDir, "strip-top-dir", false, "strip the source's root folder from the destination path, akin to \"cp dir/*\" (ex. sourcedir/subdir1/file1 copies to subdir1/file1 on destination; whereas without --strip-top-dir, it copies to sourcedir/subdir1/file1)")
+	cpCmd.PersistentFlags().BoolVar(&raw.stripTopDir, "strip-top-dir", false, "strip the source's root folder from the destination path, akin to \"cp dir/*\". Without --recursive, just copies files under the folder but does not recurse into sub-directories akin to a non-recursive sync (ex. sourcedir/subdir1/file1 copies to subdir1/file1 on destination; whereas without --strip-top-dir, it copies to sourcedir/subdir1/file1)")
```

Comment:
Can you re-order and tweak wording slightly please, to:

> strip the source's root folder from the destination path, akin to \"cp dir/*\".  E.g. sourcedir/subdir1/file1 copies to subdir1/file1 on destination; whereas without --strip-top-dir, it copies to sourcedir/subdir1/file1.  May be used with and without --recursive. Without --recursive, just copies files under the folder but does not recurse into sub-directories

Reason for suggested change is that the recursive stuff is detail, and having it before the main example would confuse users.  And the allusion to "non-recursive sync" potentially added questions ("Oh, I need to learn what sync does before I can understand this flag" etc)